src/app/fdctl/run/tiles/fd_pack.c

```diff
@@ -287,7 +287,7 @@ insert_from_extra( fd_pack_ctx_t * ctx ) {
   extra_txn_deq_remove_head( ctx->extra_txn_deq );
 
   /* Unpack the time stashed in the CUs field */
-  long receive_time = (long)((((ulong)insert->executed_cus)<<32) | ((ulong)insert->requested_cus));
+  long receive_time = (long)((((ulong)insert->pack_cu.non_execution_cus)<<32) | ((ulong)insert->pack_cu.requested_execution_cus));
```

Comment:
We should probably just add another field to the union for this use, which would make this code much less gross anyway